### ISSUE TITLE
module 'ray.serve' has no attribute 'create_backend'

### ISSUE THREAD

**Issue by Matrix-World** [ray-project/ray#15271]
I implemented the official Ray Serve case, the code(https://docs.ray.io/en/master/serve/index.html) is as follows:
------------code--------------
import ray
from ray import serve
import requests

ray.init()
serve.start()

def say_hello(request):
    return "hello " + request.query_params["name"] + "!"

serve.create_backend("my_backend", say_hello)
serve.create_endpoint("my_endpoint", backend="my_backend", route="/hello")

print(requests.get("http://127.0.0.1:8000/hello?name=serve").text)
print(ray.get(serve.get_handle("my_endpoint").remote(name="serve")))
-----------code--------------
env: ray == 1.2.0
But every time it is executed, an error will be reported,
serve.create_backend("my_backend",say_hello)
AttributeError: module'ray.serve' has no attribute'create_backend'

This confuses me. I checked the method of serve and it seems that there is no create_backend()

**Comment by volks73** [comment]
Hello, I am just started out learning this project and I came across the exact same problem and I believe I know the source of the issue. 

For me, I installed the latest release on PyPI using `pip install "ray[serve]"`, but I was reading and following along with the `master`/nightly documentation. The quickstart example/documentation is different between the latest release and the nightly release. I overlooked the nice, big warning text at the top of the documentation about the master/nightly documentation versus the pip release documentation. Either changing to the nightly release or modifying the quickstart example according to the pip release documentation resolved the issue for me.

It appears the nightly version of Ray Serve has a much more beginner friendly API.
